WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletViewModel.cs

```diff
@@ -221,5 +211,33 @@ public void OnAddWord(string word)
 
 			Suggestions.Clear();
 		}
+
+		private void ValidateMinGapLimit(IValidationErrors errors)
+		{
+			if (!int.TryParse(MinGapLimit, out int minGapLimit))
+			{
+				errors.Add(ErrorSeverity.Error, "Must be a number.");
+			}
+			else if (minGapLimit < KeyManager.AbsoluteMinGapLimit)
+			{
+				errors.Add(ErrorSeverity.Error, $"Cannot be smaller than {KeyManager.AbsoluteMinGapLimit}.");
+			}
+			else if (minGapLimit > KeyManager.MaxGapLimit)
+			{
+				errors.Add(ErrorSeverity.Error, $"Cannot be larger than {KeyManager.MaxGapLimit}.");
+			}
```

Comment:
How about this:
```
if (!int.TryParse(MinGapLimit, out int minGapLimit) || minGapLimit < KeyManager.AbsoluteMinGapLimit || minGapLimit > KeyManager.MaxGapLimit)
{
	errors.Add(ErrorSeverity.Error, $"Must be a number between {KeyManager.AbsoluteMinGapLimit} and {KeyManager.MaxGapLimit}.");
}
```

It will fix the issue described by @PulpCattel when inserting a big number.

Reply:
done.